v0/destinations/hs/transform.js

```diff
@@ -60,8 +64,19 @@ const processSingleMessage = async (message, destination, propertyMap) => {
 };
 
 // has been deprecated - using routerTransform for both the versions
-const process = event => {
-  return processSingleMessage(event.message, event.destination);
+const process = async event => {
+  const { destination } = event;
+  const mappedToDestination = get(event.message, MappedToDestinationKey);
+  let events = [];
+  events = [event];
+  if (
+    mappedToDestination &&
+    GENERIC_TRUE_VALUES.includes(mappedToDestination?.toString())
```

Comment:
```suggestion
    GENERIC_TRUE_VALUES.includes(mappedToDestination.toString())
```
we can remove `?` for all places as it is not required